plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3262,4 +3263,4 @@ dialog.bedrock.unsupported.title=Run client unsupported
 dialog.plugin_load_failed.msg1=Some plugins failed to load:
 dialog.plugin_load_failed.msg2=Certain features may not function properly. Check MCreator log files for more details.<br>\
   We recommend contacting plugin authors for more information and support.
-dialog.plugin_load_failed.title=Some plugins failed to load
\ No newline at end of file
+dialog.plugin_load_failed.title=Some plugins failed to load
```

Comment:
Please revert the new-line addition at the end of this file as it is not related to the PR

Reply:
Huh, not sure where that came from

Reply:
Done!